scripts/lib/CIME/SystemTests/system_tests_compare_two.py

```diff
@@ -282,9 +282,13 @@ def _get_output_root2(self):
 
         Assumes that self._case1 is already set to point to the case1 object
         """
-        # Since case 2 has the same name as case1 its CIME_OUTPUT_ROOT must also be different
+        # Since case2 has the same name as case1, its CIME_OUTPUT_ROOT
+        # must also be different, so that anything put in
+        # $CIME_OUTPUT_ROOT/$CASE/ is not accidentally shared between
+        # case1 and case2. (Currently nothing is placed here, but this
+        # helps prevent future problems.)
         output_root2 = os.path.join(self._case1.get_value("CIME_OUTPUT_ROOT"),
-                                    self._case1.get_value("CASE"), "case2")
+                                    self._case1.get_value("CASE"), "case2_output_root")
```

Comment:
I debated what to do about CIME_OUTPUT_ROOT. Now that we set RUNDIR and EXEROOT separately, nothing is placed here. Yet this directory is still created by the scripts. So we get a directory like `/glade/scratch/sacks/TESTNAME/case2_output_root` with nothing in it. I found this a little confusing. On the other hand, I'm nervous about leaving case2's CIME_OUTPUT_ROOT at its default value, because that opens the door for things from case2 to clobber things from case1 (in case anything is put in `$CIME_OUTPUT_ROOT/$CASE/` - since that location will be the same for case1 and case2).

As you'll see below, the solution I came up with was adding a README file to this location, so that if someone stumbles upon this directory, they'll be pointed to where they likely really want to go. But I'm open to suggestions if you feel there's a better solution.

Reply:
@billsacks , case2 will involve 3 different directions, it's RUNDIR, EXEROOT (blddir), and it's case dir. I would expect the case dir to be under the case2 output root. Is this not right?

Reply:
@jgfouca No, that's not right. case2's case directory gets nested under case1's case directory:

```python
        caseroot2 = os.path.join(caseroot1, "case2", casename2)
```

which, in general, bears no relationship to either the case1 or case2 CIME_OUTPUT_ROOT.

It's confusing, I know.

Reply:
OK, thanks.